src/cljam/io/bcf/writer.clj

```diff
@@ -22,6 +22,7 @@
   (write-variants [this variants]
     (write-variants this variants)))
 
+(def ^:private default-fileformat "VCFv4.3")
```

Comment:
Can it be also `^:const`?

Reply:
Thanks! [Fixed it](https://github.com/chrovis/cljam/compare/581438a1502bf4ab1af0d4b1ea74970b3d876320..54b217e5a5c2bab034add839e59cdae0524900aa) and rebased the branch onto master.